packages/Footer/Footer.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <section id="footer">
+    <slot>
+      <div>
+        <div class="container">
+          <nav flex row space-between wrap mobile-column-reverse>
```

Comment:
I think also `row`, `space-between`, `third`, `mbX`... are not attributes of the components. This works because you include the default theme in the storybook but what happens if the storybook doesn't include the theme? The component should still look ok.